docs/telemetry/CollectedData.template.md

```diff
@@ -0,0 +1,36 @@
+# Collected telemetry data
+
+Copied from https://github.com/ipfs-shipyard/ignite-metrics/blob/feature/metric-library/129/docs/telemetry/CollectedData.template.md
+
+<!--
+This is the default for projects using the ignite-metrics library. Each project's own `./docs/telemetry/CollectedData.md` file should contain very specific information under each of the following sections:
+
+* 'What metrics data DO we collect'
+
+-->
+## What metrics data DON'T we collect
+
+We do not collect any of the below listed items in our metric data:
+
+1. User identifiable data (names, email addresses, aliases, handles, etc.)
+2. CIDs
+3. IP addresses
+
+Countly also calls out how they protect your data at https://support.count.ly/hc/en-us/articles/360037501372-Security-Privacy-and-Access-FAQ
+
+## What metrics data DO we collect
+
+As a general rule, we collect only application data; no user data. Some examples of application data we collect are:
+
+| Metric data name | Metric feature name | What is this data used for |
+|:----------------:|---------------------|----------------------------|
+| Fill in          | Fill in             | Fill in                    |
+|                  |                     |                            |
+|                  |                     |                            |
+
+
+## Other related documents
+
+* https://github.com/ipfs-shipyard/ignite-metrics/blob/feature/metric-library/129/docs/telemetry/CollectionPolicy.md
+* https://github.com/ipfs-shipyard/ignite-metrics/blob/feature/metric-library/129/docs/telemetry/PrivacyPolicy.md
+* https://github.com/ipfs-shipyard/ignite-metrics/blob/feature/metric-library/129/docs/telemetry/FAQs.md
```

Comment:
This will need to point to `main/` prior to merging

Reply:
(just an idea for the future)
We could replace these with aliases at `ipfs.fyi` (https://github.com/ipfs/ipfs.fyi):

```
/telemetry-collection-policy https://github.com...CollectionPolicy.md
/telemetry-privacy-policy    https://github.com...PrivacyPolicy.md
/telemetry-faq               https://github.com...FAQs.md
```

Reply:
> (just an idea for the future) We could replace these with aliases at `ipfs.fyi` (https://github.com/ipfs/ipfs.fyi):
> 
> ```
> /telemetry-collection-policy https://github.com...CollectionPolicy.md
> /telemetry-privacy-policy    https://github.com...PrivacyPolicy.md
> /telemetry-faq               https://github.com...FAQs.md
> ```

I love the idea but I'm not sure exactly how this works and there doesn't seem to be enough info in the readme. Are we supposed to create a _redirects file in each repo, or just in this repo?